extractor/src/main/java/org/schabi/newpipe/extractor/services/peertube/linkHandler/PeertubeStreamLinkHandlerFactory.java

```diff
@@ -37,6 +37,12 @@ public String getId(String url) throws ParsingException, IllegalArgumentExceptio
 
     @Override
     public boolean onAcceptUrl(final String url) throws FoundAdException {
-        return url.contains("/videos/");
+        if (url.contains("playlist")) return false;
```

Comment:
maybe use `/playlist/` ? that way it would not filter `/videos/watch/696969playlist696969`

Reply:
It was crashing without it, but it was because it was at the beginning and was badly implemented. This check is not even neeeded.